### ISSUE TITLE
Input field does not reformat on text drag

### ISSUE THREAD

**Issue by granttitus** [jaridmargolin/formatter.js#20]
The input field messes up if a user highlights some text, drags, and then releases.
#### 1. Input Text

![screen shot 2013-11-13 at 8 46 56 pm](https://f.cloud.github.com/assets/458421/1537778/0bad63f6-4ccf-11e3-9527-2d5a8079d544.png)
#### 2. Highlight Section

![screen shot 2013-11-13 at 8 46 46 pm](https://f.cloud.github.com/assets/458421/1537779/0cb8d7bc-4ccf-11e3-8450-51ee00eb8dec.png)
#### 3. Drag + Release

![screen shot 2013-11-13 at 8 47 44 pm](https://f.cloud.github.com/assets/458421/1537780/0dad7d3a-4ccf-11e3-8af7-ca2210d015f6.png)

Adding an `input` event and reformatting when it's fired seemed to fix the problem. [Here's the gist](https://gist.github.com/stronglink/7459931).

**Comment by jaridmargolin** [comment]
Agreed that this is an issue, and thank you for the gist. Unfortunately, as I understand it, the 'input' event was not added until IE9. Currently formatter.js supports back to IE7. Let me look into other solutions.

**Comment by granttitus** [comment]
Alright. I'll look around for an IE7+ solution as well.
